src/lib/account/isLargeAccount.ts

```diff
@@ -0,0 +1,38 @@
+import { IS_LARGE_ACCOUNT_KEY } from "config/localStorage";
+import { useEffect } from "react";
+import { useLocalStorageSerializeKey } from "lib/localStorage";
+import { useIsLargeAccountData } from "domain/synthetics/accountStats/useIsLargeAccountData";
+import { getStorageItem } from "lib/metrics/storage";
+
+let isLargeAccount = getIsLargeAccountStoredValue();
```

Comment:
src/lib directory is more for abstract infrastructure code that not related to GMX domain

this logic looks like utils for domain/accountStats